tests/integration/s3/test_s3_cors.py

```diff
@@ -109,9 +112,9 @@ def test_cors_http_get_no_config(self, s3_bucket, snapshot, aws_client):
         response = aws_client.s3.put_object(Bucket=s3_bucket, Key=key, Body=body, ACL="public-read")
         assert response["ResponseMetadata"]["HTTPStatusCode"] == 200
 
-        # key_url = f"{_bucket_url_vhost(bucket_name=s3_bucket)}/{key}"
+        key_url = f"{_bucket_url_vhost(bucket_name=s3_bucket)}/{key}"
```

Comment:
Are the changes here in the file some leftover from debugging or intentional? If they are intentional, I think the comment should be removed.